src/test/resources/contracts/offender-bookings.avsc

```diff
@@ -0,0 +1,358 @@
+{
+  "type" : "record",
+  "namespace" : "prisons",
+  "service": "nomis",
+  "version": "1.0.0",
+  "name" : "OFFENDER_BOOKINGS",
+  "description": "A period of supervision of the identified offender while under the supervision of the Prison and/or Probation services. This may include the actual period of a custodial sentence from initial incarceration through to actual release and any follow up period of supervision during probation and subsequently. This may be for one sentence, but may also be extended by additional periods (possibly due to other sentences) which are determined during the period of supervision.",
+  "contacts" : {
+    "author": "dominic.messenger@digital.justice.gov.uk",
+    "team": "https://dsdmoj.atlassian.net/wiki/spaces/DPR/overview?homepageId=4015489077",
+    "slack": "#ask_dpr"
+  },
+  "database": {
+    "type": "oracle",
+    "version": "12",
+    "name": "CNOMT3"
+  },
+  "fields" : [
+    {
+      "name": "OFFENDER_BOOK_ID",
+      "type": "long",
+      "nullable": false,
+      "description": "The system generated identifier for an offender booking. PK",
+      "glossary-index": "offender_bookings:id",
+      "key": "primary"
+    },
+    {
+      "name": "BOOKING_BEGIN_DATE",
+      "type": {
+        "type": "int",
+        "logicalType": "date"
+      },
+      "nullable": true,
+      "description": "The date the booking was created."
+    },
+    {
+      "name": "BOOKING_END_DATE",
+      "type": {
+        "type": "int",
+        "logicalType": "date"
+      },
+      "nullable": false,
+      "description": "The date the booking was closed."
+    },
+    {
+      "name": "BOOKING_NO",
+      "type": "string",
+      "nullable": false,
+      "description": "The identifier for the booking which is visible to the user."
+    },
+    {
+      "name": "OFFENDER_ID",
+      "type": "long",
+      "nullable": true,
+      "glossary-index": "offender:id",
+      "description": "The unique identifier for an offender."
+    },
+    {
+      "name": "AGY_LOC_ID",
+      "type": "string",
+      "nullable": true,
+      "glossary-index": "location:id",
+      "description": "The location residing within an agency."
+    },
+    {
+      "name": "LIVING_UNIT_ID",
+      "type": "long",
+      "nullable": true,
+      "glossary-index": "living_unit:id",
```

Comment:
Isn't this the internal-locations:id ? Should the internal-locations:id actually be called living-unit:id ?  
Please clarify with Michael/Zahoor and ensure this schema and internal-locations schema are the same.

Reply:
It's called internal-location:id in the agency_internal_locations. This needs to be the SAME.